inst/tests/tests.Rraw

```diff
@@ -27,6 +27,7 @@ if (exists("test.data.table", .GlobalEnv, inherits=FALSE)) {
   brackify = data.table:::brackify
   chmatchdup = data.table:::chmatchdup
   compactprint = data.table:::compactprint
+  CsubsetDT = data.table:::CsubsetDT
```

Comment:
and this is caused by what?

Reply:
https://github.com/Rdatatable/data.table/blob/master/.dev/CRAN_Release.cmd#L111-L112

I'm not quite sure the point of it, but if we're blocking it in the source we may as well block it in tests:

https://github.com/Rdatatable/data.table/pull/4278/files#diff-595e663fedb004ffced209a139805080L6411

Reply:
make sense.
fyi @mattdowle

Reply:
BTW IIUC this test is related to making sure we're using the C API correctly. If we're not registering our routines "properly" we'd have to refer to them as a string, and use `package` in the `.Call` call, I think the way we're doing it using `dyn.load` adds it to the symbol table so we can refer by symbol which is preferred. So the quotation check is about registering routines.